bin/analyze_2lss_1tau.cc

```diff
@@ -2460,6 +2464,13 @@ TMVAInterface mva_Hjj_tagger(mvaFileName_Hjj_tagger, mvaInputVariables_Hjj_tagge
   }
   std::cout << std::endl;
 
+//--- manually write histograms to output file
+  fs.file().cd();
+  histogram_analyzedEntries->Write();
+  histogram_selectedEntries->Write();
```

Comment:
By removing these lines the second cycle of 'selectedEntries' and 'analyzedEntries' histograms disappear from the output. It's total mystery to me but it somehow works. I propose that you remove them, and when you implement the logic in other channels, you don't add these two lines.

Reply:
Just to be clear, I meant removing
```cpp
histogram_analyzedEntries->Write();
histogram_selectedEntries->Write();
```